src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Utility/RemoteErrorUtility.cs

```diff
@@ -60,6 +60,10 @@ public static RemoteError ToRemoteError(Exception exception)
                         .ToArray();
                 }
             }
+            else
+            {
+                projectContextLogMessage = exception.ToString();
```

Comment:
Sounds like this resolve the null `message` issue I found.

Reply:
Yup